content/geoip/license-key-validation-api.mdx

```diff
@@ -0,0 +1,97 @@
+used for compliance purposes. By using our License Key Validation API in
+conjunction with these tools, you can automate the process of finding and
+verifying MaxMind license keys in your codebase. This can help prevent license
+keys accidentally ending up in unexpected places and enhance the security of
+your applications.
+
+## Request:
+
+### Request URI
+
+API calls should be made with an HTTP POST request to:
+
+`https://secret-scanning.maxmind.com/secrets/validate-license-key`
+
+### Authorization/Security
+
+There is no authorization needed for this endpoint. The request is made using
+the license key itself.
+
+### Request Body
+
+The Content Type of the request must be `application/x-www-form-urlencoded`. No
+other content types are accepted at the moment.
+
+The request body may include the following field:
+
+| Key           | Value Type | Description                                                                       |
+| ------------- | ---------- | --------------------------------------------------------------------------------- |
+| `license_key` | string     | The `license_key` field should contain the license key that you want to validate. |
+
+Here is an example of how to make the request using curl:
+
+```bash
+curl https://secret-scanning.maxmind.com/secrets/validate-license-key \
+    -F license_key="your_license_key_here"
+```
+
+Please replace "your_license_key_here" with the actual license key you want to
+validate.
+
+## Response
+
+HTTP status codes are used to relay success and error messages. A successful
+POST will return a 204 (No Content) status code.
+
+When the server returns an error (4xx or 5xx), the response may include a JSON
+document in the body. This document is a single object with the keys `code` and
+`error`. The `code` field is a static error code for machine use. The value of
+any given code will never change, though codes can be added or removed. The
+`error` field is a human-readable description of the error and may change at any
+time.
+
+### Response Body (for successful requests)
+
+When the license key is successfully validated, the server will return a status
+of `204 No Content`. This means that the request was successful.
+
+### Response Body (for unauthorized license keys)
+
+When the license key is in the correct format but could not be validated in our
+database, the server will return a JSON object with the following structure:
+
+```json
+{
+  "code": "AUTHORIZATION_INVALID",
+  "error": "Your account ID or license key could not be authenticated."
+}
+```
+
+The HTTP status will be `401 Unauthorized`.
+
+### Response Body (for unsuccessful requests)
+
+In the event of an error occuring such as a bad request that has a license key
+with a bad format, the server will return a JSON object with the following
+structure:
+
+```json
+{
+  "code": "LICENSE_KEY_INVALID",
+  "error": "'foo-bad-license-key' is not a valid license_key. when calling /secrets/validate-license-key"
```

Comment:
We recently improved the error message here given the punctuation issue. Assuming we keep this example, could we use the new message?